controllers/api/v1/foods_controller.js

```diff
@@ -13,7 +13,7 @@ async function index(req, res) {
     res.status(200).send({ foods })
   } else {
     res.setHeader("Content-Type", "application/json");
-    res.status(200).send({ foods })
+    res.status(200).send({ "foods": [] })
```

Comment:
Another way to fix this might be to set foods as an empty array above the if statement.
But hard coding such a small thing isn't an issue here since it's going to be fairly static.